src/operators_dense.jl

```diff
@@ -20,6 +20,9 @@ end
 Operator{BL,BR}(basis_l::BL,basis_r::BR,data::T) where {BL,BR,T} = Operator{BL,BR,T}(basis_l,basis_r,data)
 Operator(basis_l::BL,basis_r::BR,data::T) where {BL,BR,T} = Operator{BL,BR,T}(basis_l,basis_r,data)
 Operator(b::Basis,data) = Operator(b,b,data)
+Operator(qet1::Ket, qetva::Ket...) = Operator(qet1.basis, GenericBasis(length(qetva)+1), qet1, qetva...)
+Operator(basis_r::Basis,qet1::Ket,qetva::Ket...) = Operator(qet1.basis, basis_r, qet1, qetva...)
+Operator(basis_l::BL,basis_r::BR,qet1::Ket,qetva::Ket...) where {BL,BR} = Operator{BL,BR}(basis_l, basis_r, hcat(qet1.data, getfield.(qetva,:data)...))
```

Comment:
While this will work fine, compilation might be very slow for some use cases, such as constructing unitaries from sets of Kets. If the dimension of the space is large, there will be a large number of arguments to this function. Could you add a `Operator(qets::AbstractVector{<:Ket})` or similar and use `reduce(hcat, getfield.(qets, :data)))` internally to avoid the splat?